plugins/parodos/src/models/project.ts

```diff
@@ -8,17 +8,26 @@ const projectStatus = z.union([
   z.literal('COMPLETED'),
 ]);
 
+const accessRole = z.union([
+  z.literal('Owner'),
+  z.literal('Developer'),
+  z.literal('Admin'),
+]);
+
 export type ProjectStatus = z.infer<typeof projectStatus>;
 
 export const projectSchema = z.object({
   id: z.string(),
   name: z.string(),
   description: z.string().nullable(),
```

Comment:
```suggestion
  description: z.string().nullable().optional(),
```
Needs to be optional now as the API is no longer returning `null` for description.